scene/main/multiplayer_peer.cpp

```diff
@@ -201,6 +201,9 @@ void MultiplayerPeerExtension::_bind_methods() {
 	GDVIRTUAL_BIND(_get_packet_script)
 	GDVIRTUAL_BIND(_put_packet_script, "p_buffer");
 
+	GDVIRTUAL_BIND(_get_packet_channel);
+	GDVIRTUAL_BIND(_get_packet_mode);
```

Comment:
I think we are also missing:

`GDVIRTUAL_BIND(_is_server_relay_supported);`

which I forgot to add too.

Reply:
Ah, yes, thanks! I've added it in my latest push